core/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+
+plugins {
+    id("buildlogic.java-conventions")
+    `maven-publish`
+    id("com.gradleup.shadow")
+}
+
+tasks.named<ShadowJar>("shadowJar") {
+    archiveFileName = "Community.jar"
+    archiveClassifier.set("")
+    destinationDirectory = rootProject.projectDir.resolve("build/libs")
+
+    minimize()
+
+    dependencies {
+        exclude(dependency("org.jetbrains:annotations"))
+    }
+
+    exclude("META-INF/**")
+}
+
+publishing {
+    publications.create<MavenPublication>("community") {
+        groupId = project.group as String
+        artifactId = project.name
+        version = project.version as String
+
+        artifact(tasks["shadowJar"])
+    }
+    repositories {
+        maven {
+            name = "pgm-repo-snapshots"
+            url = uri("https://repo.pgm.fyi/snapshots")
```

Comment:
Publishing did not appear to be set up for the workflow previously; this likely does not work with the current workflow setup. 

PGM publishes GitHub packages which seems appropriate to do here until publishing to `repo.pgm.fyi` can be set up later.